### PR TITLE
Add Multiplecascade to TreeGrid selection sample

### LINKED ISSUE #2454: Modify igxTreeGrid Row Selection sample in order to add cascading selection
### Description  
Modify the existing [igxTreeGrid Row Selection sample](https://www.infragistics.com/products/ignite-ui-angular/angular/components/treegrid/row-selection) so that an option for the [new multiple cascade selection](https://github.com/IgniteUI/igniteui-angular/issues/8040) is added.

- add button in the button group for cascade mode
- add snackbar message corresponding to the new mode

### PR DESCRIPTION
**Pull request by teodosiah**
Closes #2454 

Modified the igxTreeGrid selection sample by adding a cascade selection.

### CODE REVIEW

**Review by StefanIvanov — APPROVED** [review]

**Review by mmart1n — APPROVED** [review]

**Review by kacheshmarova — APPROVED** [review]

**Comment by DiyanDimitrov** [comment]
Adding `do not merge` label since this can't be merged before the feature is merged and released in an alpha.

**Pull request merged**
